generated/python/grpc-google-cloud-logging-v2/setup.py

```diff
@@ -33,11 +33,11 @@
     'Programming Language :: Python :: 3.5',
     'Programming Language :: Python :: Implementation :: CPython',
   ],
-  description='GRPC library for the google-logging-v2 service',
+  description='GRPC library for the Google Logging service',
```

Comment:
Is this change auto-generated? It should be "Stackdriver Logging". IIUC, your plan is to eventually pull this from the service config `title` (e.g., https://github.com/googleapis/googleapis/blob/master/google/logging/logging.yaml#L6).